packages/repositories.config

```diff
@@ -1,5 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <repositories>
+  <repository path="../Octokit.Reactive/packages.config" />
```

Comment:
:lipstick: this just seems to be some mixed up `\` and `/` path separators - can we unify these?